Downloading P5 files from Xubmit in admin page is not successful in the local environment

I tried to update my local P5 files from Xubmit on the admin page (http://localhost:8080/admin), but I encountered some errors.

When I tried to update the authoritative bibliography, I got this message:
```
com.xmlcalabash.core.XProcException: Could not load http://algernon.dlib.indiana.edu:8080/xubmit/rest/repository/newtonbib/CHYM000001.xml (file:/[MY LOCATION]/newton_chymistry/xproc/administration.xpl) dtd-validate=false
	at com.xmlcalabash.core.XProcException.stepError(XProcException.java:184)
	at com.xmlcalabash.library.Load.run(Load.java:82)
	at com.xmlcalabash.runtime.XAtomicStep.run(XAtomicStep.java:389)
	at com.xmlcalabash.runtime.XPipeline.doRun(XPipeline.java:236)
	at com.xmlcalabash.runtime.XPipeline.run(XPipeline.java:136)
	at com.xmlcalabash.runtime.XPipelineCall.run(XPipelineCall.java:94)
	at com.xmlcalabash.runtime.XCompoundStep.run(XCompoundStep.java:262)
	at com.xmlcalabash.runtime.XChoose.run(XChoose.java:135)
	at com.xmlcalabash.runtime.XPipeline.doRun(XPipeline.java:236)
	at com.xmlcalabash.runtime.XPipeline.run(XPipeline.java:136)
	at com.conaltuohy.xprocz.XProcZServlet$RunnablePipeline.run(XProcZServlet.java:160)
	at com.conaltuohy.xprocz.XProcZServlet.service(XProcZServlet.java:524)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:668)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:615)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:818)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1627)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
```

Also, when I attempted to download the manuscript files, I got this message:
```
Error err:FOJS0001net.sf.saxon.s9api.SaxonApiException: Invalid JSON input: Unexpected character '<' (\u3c) at position 0
	at net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:606)
	at com.xmlcalabash.library.XSLT.run(XSLT.java:254)
	at com.xmlcalabash.runtime.XAtomicStep.run(XAtomicStep.java:389)
	at com.xmlcalabash.runtime.XPipeline.doRun(XPipeline.java:236)
	at com.xmlcalabash.runtime.XPipeline.run(XPipeline.java:136)
	at com.xmlcalabash.runtime.XPipelineCall.run(XPipelineCall.java:94)
	at com.xmlcalabash.runtime.XCompoundStep.run(XCompoundStep.java:262)
	at com.xmlcalabash.runtime.XChoose.run(XChoose.java:135)
	at com.xmlcalabash.runtime.XPipeline.doRun(XPipeline.java:236)
	at com.xmlcalabash.runtime.XPipeline.run(XPipeline.java:136)
	at com.conaltuohy.xprocz.XProcZServlet$RunnablePipeline.run(XProcZServlet.java:160)
	at com.conaltuohy.xprocz.XProcZServlet.service(XProcZServlet.java:524)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:668)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:615)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:818)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1627)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: net.sf.saxon.trans.XPathException: Invalid JSON input: Unexpected character '<' (\u3c) at position 0
	at net.sf.saxon.ma.json.JsonParser.error(JsonParser.java:401)
	at net.sf.saxon.ma.json.JsonParser.invalidJSON(JsonParser.java:414)
	at net.sf.saxon.ma.json.JsonParser.access$000(JsonParser.java:26)
	at net.sf.saxon.ma.json.JsonParser$JsonTokenizer.readToken(JsonParser.java:563)
	at net.sf.saxon.ma.json.JsonParser$JsonTokenizer.next(JsonParser.java:453)
	at net.sf.saxon.ma.json.JsonParser.parse(JsonParser.java:68)
	at net.sf.saxon.ma.json.ParseJsonFn.parse(ParseJsonFn.java:89)
	at net.sf.saxon.ma.json.JsonDoc.call(JsonDoc.java:86)
	at net.sf.saxon.expr.FunctionCall.iterate(FunctionCall.java:547)
	at net.sf.saxon.expr.instruct.GlobalVariable.getSelectValue(GlobalVariable.java:643)
	at net.sf.saxon.expr.instruct.GlobalVariable.actuallyEvaluate(GlobalVariable.java:724)
	at net.sf.saxon.expr.instruct.GlobalVariable.evaluateVariable(GlobalVariable.java:692)
	at net.sf.saxon.expr.GlobalVariableReference.evaluateVariable(GlobalVariableReference.java:126)
	at net.sf.saxon.expr.VariableReference.iterate(VariableReference.java:536)
	at net.sf.saxon.expr.ItemChecker.iterate(ItemChecker.java:227)
	at net.sf.saxon.expr.ForExpression.iterate(ForExpression.java:399)
	at net.sf.saxon.expr.ItemChecker.iterate(ItemChecker.java:227)
	at net.sf.saxon.expr.ForExpression.iterate(ForExpression.java:399)
	at net.sf.saxon.expr.FilterExpression.iterate(FilterExpression.java:1139)
	at net.sf.saxon.expr.instruct.ForEach.processLeavingTail(ForEach.java:434)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.LetExpression.processLeavingTail(LetExpression.java:709)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:505)
	at net.sf.saxon.Controller.transformDocument(Controller.java:2411)
	at net.sf.saxon.Controller.transform(Controller.java:1979)
	at net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:596)
	... 34 more
```

Judging from the messages, it does not seem to be due to my local side issues. If anyone knows what is happening here, please let me know.

--- a/xproc/administration.xpl
+++ b/xproc/administration.xpl
@@ -75,7 +75,7 @@
 	<p:declare-step name="download-bibliography" type="chymistry:download-bibliography">
 		<p:input port="source"/>
 		<p:output port="result"/>
-		<p:load href="http://algernon.dlib.indiana.edu:8080/xubmit/rest/repository/newtonbib/CHYM000001.xml"/>
+		<p:load href="https://textproc.dlib.indiana.edu/xubmit/rest/repository/newtonbib/CHYM000001.xml"/>
 		<p:store href="../p5/CHYM000001.xml"/>
 		<p:identity>
 			<p:input port="source">
@@ -98,7 +98,7 @@
 		<!-- algernon.dlib.indiana.edu:8080 -->
 		<!-- host was textproc.dlib.indiana.edu -->
 		<p:option name="dc-coverage-regex"/>
-		<p:variable name="xubmit-base-uri" select=" 'http://textproc.dlib.indiana.edu/xubmit/rest/repository/newtonchym/' "/>
+		<p:variable name="xubmit-base-uri" select=" 'https://textproc.dlib.indiana.edu/xubmit/rest/repository/newtonchym/' "/>
 		<p:xslt name="manifest">
 			<p:with-param name="base-uri" select="$xubmit-base-uri"/>
 			<p:with-param name="dc-coverage-regex" select="$dc-coverage-regex"/>

--- a/xproc/convert-to-p5.xpl
+++ b/xproc/convert-to-p5.xpl
@@ -12,7 +12,7 @@
 		<p:input port="source"/>
 		<p:output port="result"/>
 		<p:option name="dc-coverage-regex"/>
-		<p:variable name="xubmit-base-uri" select=" 'http://algernon.dlib.indiana.edu:8080/xubmit/rest/repository/newton/' "/>
+		<p:variable name="xubmit-base-uri" select=" 'https://textproc.dlib.indiana.edu/xubmit/rest/repository/newton/' "/>
 		<p:xslt name="manifest">
 			<p:with-param name="base-uri" select="$xubmit-base-uri"/>
 			<p:with-param name="dc-coverage-regex" select="$dc-coverage-regex"/>

--- a/xslt/html-directory-listing.xsl
+++ b/xslt/html-directory-listing.xsl
@@ -4,7 +4,7 @@
 	xmlns:c="http://www.w3.org/ns/xproc-step" 
 	xmlns="http://www.w3.org/1999/xhtml">
 	
-	<xsl:variable name="xubmit-base-url" select=" 'http://algernon.dlib.indiana.edu:8080/xubmit/rest/repository/newton/' "/>
+	<xsl:variable name="xubmit-base-url" select=" 'https://textproc.dlib.indiana.edu/xubmit/rest/repository/newton/' "/>
 
 	<xsl:template match="/c:directory">
 		<html>

